### PR TITLE
Doc: merge remarks from https://github.com/SNCFdevelopers/API-trains-sncf/pull/11

### PR DESCRIPTION
**Pull request by stifoon**
https://github.com/SNCFdevelopers/API-trains-sncf/pull/11

### CODE REVIEW

**Comment by stifoon** [comment]
Jenkins, test this please.

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -310,9 +310,8 @@ chapter
 
 Also known as `/pt_objects` service.
 
-This api search in public transport objects via their names. It's a kind
-of magical autocomplete on public transport data. It returns, in
-addition of classic objects, a collection of [places](#place).
+This endpoint allows you to search in public transport objects using their names. It's a kind
+of magical autocomplete on public transport data. It returns a collection of [places](#place).
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: returns a collection of pt_objects, no?

File: documentation/slate/source/includes/apis.md [hunk 2]
```diff
@@ -411,9 +410,9 @@ Response example for :
 
 Also known as `/places` service.
 
-This api search in all geographical objects via their names. It returns,
-in addition of classic objects, a collection of [places](#place) . This api is
-very useful to make some autocomplete stuff.
+This endpoint allows you to search in all geographical objects using their names, returning
+a [places](#place) collection.
+It is very useful to make some autocomplete stuff.
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: Are you trying to say that it is usefull to help understand what place the user is looking for, even if he has mittens? ;)
-> This sentence is not so clear to me.

File: documentation/slate/source/includes/nearby.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ You can use these APIs (click on them for details):
 -   **[Coverage](#coverage)** : List of the region covered by navitia
 -   **[Public transportation objects](#pt-ref)** : List of the public transport
     objects of a region
--   **[Places nearby](#places-nearby)** : List of objects near an object or a coord
+-   **[Places nearby](#places-nearby)** :  Return a list of objects near an object or using longitude and latitude
```
Comment (pbougue) [inline documentation/slate/source/includes/nearby.md]: `Return` (maybe returns) everywhere, or `list` everywhere?

File: documentation/slate/source/includes/apis.md [hunk 4]
```diff
@@ -411,8 +411,9 @@ Response example for :
 Also known as `/places` service.
 
 This endpoint allows you to search in all geographical objects using their names, returning
-a [places](#place) collection.
-It is very useful to make some autocomplete stuff.
+a [place](#place) collection.
+
+It is very useful to make some [autocomplete](https://en.wikipedia.org/wiki/Autocomplete) stuff, even if the user has mittens.
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: minor: For me, autocomplete IS "understanding what the user wants, even if he has mittens". It's not a particular property of our autocomplete.

**Comment by stifoon** [comment]
OK for u @pbougue ?

**Pull request merged**